### PR TITLE
fix: (Platform) add fix for screen reader to read the thumbnail label and selected thumbnail image for Thumbnail component.

### PR DESCRIPTION
**Pull request by manu-kr**
#### Please provide a link to the associated issue.
Closes https://github.com/SAP/fundamental-ngx/issues/5078

#### Please provide a brief summary of this pull request.
This PR changes contains fix for screen reader to read the thumbnail ge label and selected thumbnail image for Thumbnail component.

#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist
- [x] Run npm run build-pack-library and test in external application

Documentation checklist:
- [NA] update `README.md`
- [NA] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
-  [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

File: libs/platform/src/lib/components/thumbnail/thumbnail-image/thumbnail-image.component.ts [hunk 1]
```diff
@@ -26,6 +26,8 @@ export class ThumbnailImageComponent implements OnChanges, OnInit {
     @Output()
     thumbnailClicked: EventEmitter<Media> = new EventEmitter();
 
+    moreLabel: string;
```
Comment (kavya-b) [inline libs/platform/src/lib/components/thumbnail/thumbnail-image/thumbnail-image.component.ts:29]: please add some doc/description for this field

Comment (manu-kr) [reply libs/platform/src/lib/components/thumbnail/thumbnail-image/thumbnail-image.component.ts]: Modeified.

**Review by kavya-b — APPROVED** [review]
LGTM. Please check travis failing.

**Review by KevinOkamoto — APPROVED** [review]

**Review by InnaAtanasova — APPROVED** [review]

**Comment by Seamoo13** [comment]
- [ ] case where screen reader says 'not selected' when you select a thumbnail using mouse click
**STR**
- enable a screen reader with log printing
- go to the standard thumbnail example
- click on the 1st item
- check screen reader output (correct, says item selected)
- click on the 3rd item

**Actual results:** screen reader says the item is not selected, even though you just selected it

**Expected results:** if item is selected, screen reader will say item selected

**note:** works fine for tab and enter key



https://user-images.githubusercontent.com/47522152/119616243-0dc61300-be09-11eb-8086-328317d3a25f.mp4


- [ ] In the issue's attached doc, the issue number 3 says the list count should be read. I see the list count on the 1st item, but the other items don't say item 2 of 5, 3 of 5 as the document suggests to do. Was it discussed to only give the total list count?

**Comment by manu-kr** [comment]
> * [ ]  case where screen reader says 'not selected' when you select a thumbnail using mouse click
>   **STR**
> * enable a screen reader with log printing
> * go to the standard thumbnail example
> * click on the 1st item
> * check screen reader output (correct, says item selected)
> * click on the 3rd item
> 
> **Actual results:** screen reader says the item is not selected, even though you just selected it
> 
> **Expected results:** if item is selected, screen reader will say item selected
> 
> **note:** works fine for tab and enter key
> 
> 
> 
>  Peek.2021-05-26.09-51.mp4 
> * [ ]  In the issue's attached doc, the issue number 3 says the list count should be read. I see the list count on the 1st item, but the other items don't say item 2 of 5, 3 of 5 as the document suggests to do. Was it discussed to only give the total list count?
>   

>@Seamoo13 , Have a question on this behaviour and needs to discuss with QA team on this.

**Comment by manu-kr** [comment]
This issue is on hold.

**Pull request closed without merging**